source/docs/software/driverstation/driver-station-best-practices.rst

```diff
@@ -15,7 +15,7 @@ Prior To Departing For The Competition
 3. Consider used laptops rather than new. The FRC\ |reg| Driver Station and dashboard software uses very few system resources, so you don't need to buy a new laptop -- instead, buy a cheap 4-5 year old used one. You might even get one donated by a used computer store in your area.
```

Comment:
While we're here, this should caution against buying laptops too old to run Windows 11 (e.g. Intel 8th generation or newer).